skyportal/utils/moving_objects.py

```diff
@@ -0,0 +1,505 @@
+import urllib.parse
+from datetime import datetime
+from io import StringIO
+
+import numpy as np
+import pandas as pd
+import requests
+import sqlalchemy as sa
+from astropy.coordinates import AltAz, SkyCoord, get_body
+from astropy.time import Time
+
+from baselayer.log import make_log
+from skyportal.models import (
+    InstrumentField,
+    InstrumentFieldTile,
+)
+from skyportal.utils.calculations import (
+    dms_to_deg,
+    get_airmass,
+    great_circle_distance_vec,
+    hms_to_deg,
+    radec_to_healpix,
+)
+
+from .cache import Cache, dict_to_bytes
+
+cache_dir = "cache/moving_object_ephemeris"
+cache = Cache(
+    cache_dir=cache_dir,
+    max_items=100,
+)
+
+log = make_log("api/moving_object")
+
+# next we define some constants
+BASE_URL = "https://ssd.jpl.nasa.gov"
+COLUMN_NAMES = [
+    "time",
+    "solar_presence",
+    "lunar_presence",
+    "ra",
+    "dec",
+    "airmass",
+    "mag_ex",
+    "mag_ap",
+    "drop1",
+    "drop2",
+]
+
+
+def find_obj(obj_name: str):
```

Comment:
Could name this like find_obj_horizons to disambiguate.

Reply:
good point